src/phoenix/session/session.py

```diff
@@ -331,8 +331,25 @@ def query_spans(
         root_spans_only: Optional[bool] = None,
         project_name: Optional[str] = None,
     ) -> Optional[Union[pd.DataFrame, List[pd.DataFrame]]]:
+        """
+        Queries the spans in the project based on the provided parameters.
+
+        Args:
+            queries: Variable-length argument list of SpanQuery objects
+            representing the queries to be executed. start_time: Optional
+            datetime object representing the start time of the query. stop_time:
+            Optional datetime object representing the stop time of the query.
+            root_spans_only: Optional boolean indicating whether to include only
+            root spans in the results. project_name: Optional string
+            representing the name of the project to query. Defaults to the project name
+            set in the environment variable `PHOENIX_PROJECT_NAME` or 'default' if not set.
```

Comment:
All of the arguments have been collapsed. I usually put a line in between so that you can use the auto-wrap.

Reply:
The default project name in the docstring does not match the default project name above.

Reply:
Ah yeah re-wrap messed this up